projects/plugins/crm/includes/class-learn-menu.php

```diff
@@ -444,7 +441,11 @@ public function render_generic_learn_menu(
 	          			){
 	          				$sidebar = true;
 	          			} ?>
-					<div class="ui button brown tiny learn" id="learn"><i class="fa fa-graduation-cap" aria-hidden="true"></i> <?php esc_html_e( "Learn", 'zero-bs-crm' ); ?></div>
+					<div class="ui tiny learn button" id="learn">
+					<svg class="gridicon gridicons-info-outline needs-offset" height="20" width="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+					<g><path d="M13 9h-2V7h2v2zm0 2h-2v6h2v-6zm-1-7c-4.41 0-8 3.59-8 8s3.59 8 8 8 8-3.59 8-8-3.59-8-8-8m0-2c5.523 0 10 4.477 10 10s-4.477 10-10 10S2 17.523 2 12 6.477 2 12 2z"></path></g>
+					</svg>
```

Comment:
Could we use a `svg` file instead?

Reply:
Hmm I'm not too sure, could we include our custom classes with an imported SVG file?

I tried adding SVGs with the raw data, but using `wp_remote_retrieve_body( wp_remote_get( esc_url( plugin_dir_url( __FILE__ ) . '../i/external-link.svg' ) ) )` seems like a complicated way to get that data (this is as far as I've got looking at this this afternoon anyway).

Reply:
Three drive-by comments:

* SVG has long-standing lack of native support in WP: https://core.trac.wordpress.org/ticket/24251
* Never use `..` in paths/URLs...path traversal attacks are a real threat.
* We do use it for MailPoet: https://github.com/Automattic/jetpack/blob/417605f68e603bc9ad3139d58f800d2ba3745b62/projects/plugins/crm/modules/mailpoet/admin/mailpoet-hub/main.page.php#L81

Reply:
> We do use it for MailPoet:

😮 Missed that. I now notice in VS code the icons next to coded SVG files are different which is helpful too.

> `<img id="jpcrm-mailpoet-mailpoet-logo" src="<?php echo esc_url( ZEROBSCRM_URL ); ?>i/mailpoet-logo.svg" alt="MailPoet" /> `

Brilliant, thank you! That will also help solve `Never use .. in paths/URLs...path traversal attacks are a real threat.
` (the use of `ZEROBSCRM_URL` in there particularly). Thanks for mentioning that about path traversals!


> We already have access to all FontAwesome icons if that's an alternative: [fontawesome.com/search?q=info&o=r](https://fontawesome.com/search?q=info&o=r)

I don't believe so, not if we want to follow the Jetpack style guide.